CHANGELOG.md

```diff
@@ -10,8 +10,12 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Added
 
+- Implemented box.schema.user operations requests and sugar interface.
+
 ### Changed
 
+- Box Info method now requires context and implements CallRequest type instead custom baseRequest.
```

Comment:
This is a breaking change. We need an another method and keep a backward compatibility here. You could create an issue to delete an old method in the next major release.

Reply:
Removed